### PR TITLE
Add test emulating real-world workflow for corp_text_rbindlist

### PR DESCRIPTION
**Pull request by edmondac**

### CODE REVIEW

File: tests/testthat/test-corp_text.R [hunk 1]
```diff
@@ -30,8 +30,75 @@ test_that("corp_text_rbindlist", {
...
+
+    # This token appears twice...
+    expect_equal(tokens[token == "Holmes", "start"]$start, c(14, 749))
+
+    # This token appears four times...
+    expect_equal(tokens[token == "and", "start"]$start, c(195, 649, 789, 1166))
+
+    # Now check the boundary markers
+    expect_equal(tokens[token == "occupation", "end"]$end, 832)  #  immediately before a boundary
+    expect_equal(tokens[token == "Well", "start"]$start, 688)  #  immediately after a boundary
```
Comment (mike-allaway) [inline tests/testthat/test-corp_text.R]: very minor, but 2 white spaces after the hashtag for the comment, rather than 1. (expected 2 before, but 1 after)

**Comment by mike-allaway** [comment]
Hi @edmondac just the one minor change of whitespace in comments and then I'm happy to approve and merge once that's resolved.

File: tests/testthat/test-corp_text.R [hunk 2]
```diff
@@ -30,8 +30,75 @@ test_that("corp_text_rbindlist", {
...
+    expect_equal(tokens[type == "penang", "end"]$end, 365)
+
+    # This token appears twice...
+    expect_equal(tokens[token == "Holmes", "start"]$start, c(14, 749))
+
+    # This token appears four times...
+    expect_equal(tokens[token == "and", "start"]$start, c(195, 649, 789, 1166))
+
+    # Now check the boundary markers
+    expect_equal(tokens[token == "occupation", "end"]$end, 832)  #  immediately before a boundary
```
Comment (edmondac) [inline tests/testthat/test-corp_text.R]:
```suggestion
    expect_equal(tokens[token == "occupation", "end"]$end, 832)  # immediately before a boundary
```

File: tests/testthat/test-corp_text.R (hunk 1 above)
Comment (edmondac) [inline tests/testthat/test-corp_text.R]:
```suggestion
    expect_equal(tokens[token == "Well", "start"]$start, 688)  # immediately after a boundary
```

Comment (edmondac) [reply tests/testthat/test-corp_text.R]: Done

**Review by mike-allaway — APPROVED** [review]
lgtm

**Pull request merged**